test/extended/images/info.go

```diff
@@ -23,7 +23,7 @@ var _ = g.Describe("[sig-imageregistry][Feature:ImageInfo] Image info", func() {
 
 	oc = exutil.NewCLIWithPodSecurityLevel("image-info", admissionapi.LevelBaseline)
 
-	g.It("should display information about images", func() {
+	g.It("should display information about images [apigroup:image.openshift.io]", func() {
```

Comment:
Does the image apigroup requirement come from `cliPodWithPullSecret`? If so, is builder SA also implying build apigroup?

Reply:
The image this test uses holds `oc` binary in it and it comes from the internal registry. This is only working when this apigroup is installed.